client/src/utils/storage.ts

```diff
@@ -0,0 +1,11 @@
+const saveItem = (key: string, value: { mic: boolean; speaker: boolean; cam: boolean }) => {
+  localStorage.setItem(key, JSON.stringify(value));
+};
+
+const loadItem = (key: string) => {
+  const data = localStorage.getItem(key);
```

Comment:
```suggestion
  sessionStrorage.setItem(key, JSON.stringify(value));
};

const loadItem = (key: string) => {
  const data = sessionStrorage.getItem(key);
```
로컬 스토리지 사용시 여러 세션에서 스토리지를 공유해 서로의 상태에 영향을 줄 수 있을 것 같습니다.
세션 스토리지를 사용하는 것이 좋을 것 같습니다!

Reply:
넵 반영했습니다!